Python/pywarpx/picmi.py

```diff
@@ -1393,6 +1402,9 @@ def initialize_inputs(self, solver):
             pywarpx.eb2.stl_center = self.stl_center
             pywarpx.eb2.stl_reverse_normal = self.stl_reverse_normal
 
+        if self.cover_multiple_cuts:
```

Comment:
This can be a direct assignment
```
pywarpx.eb2.cover_multiple_cuts = self.cover_multiple_cuts
```
The interface already handles the setting of booleans.